.nvmrc

```diff
@@ -0,0 +1 @@
+v8.2.0
```

Comment:
This should just be `8` should it not? That way the latest 8.x.x version is used by way of `nvm use 8`